modules/kubernetes-addons/cloudnative-pg/main.tf

```diff
@@ -0,0 +1,27 @@
+locals {
+  name = "cnpg"
+
+  # https://github.com/cloudnative-pg/charts/blob/main/charts/cloudnative-pg/Chart.yaml
+  default_helm_config = {
```

Comment:
move these down to the module declaration - no need to define in a local here

Reply:
we may need it for the future update (like version of the charts)